### PR TITLE
Feat : Added Auth to app.

### PR DESCRIPTION
**Pull request by syed-shaida-hussain**
Added authentication to the app. Please review.
What not to check : CSS files.

### CODE REVIEW

File: src/Services/login-service.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+const signinService = async ({ username, password }) => {
+  try {
+    const { data } = await axios.post('/api/auth/login', {
+      username: username,
+      password: password
+    });
+    console.log(data.foundUser);
+    return data;
+  } catch (e) {
+    alert('user not found');
```
Comment (vidhi-thakur) [inline src/Services/login-service.js:12]: Just a suggestion: For any error, alerts are not really used nowadays as it is not considered good UX. Consider using `console.error`

File: src/Services/signup-service.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+const signupService = async ({ username, password, name, email }) => {
+  try {
+    const { data } = await axios.post('/api/auth/signup', {
+      username: username,
+      password: password,
+      name: name,
+      email: email
+    });
+    console.log(data.createdUser);
```
Comment (vidhi-thakur) [inline src/Services/signup-service.js:11]: do consider to do a code clean up after the development and remove all unnecessary console logs

**Review by vidhi-thakur — APPROVED** [review]
The code formatting is good. Left a few suggestions, other than that code looks good to me.

**Review by simran61 — APPROVED** [review]
Looks good

File: src/Services/login-service.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
+
+const signinService = async ({ username, password }) => {
+  try {
+    const { data } = await axios.post('/api/auth/login', {
+      username: username,
+      password: password
+    });
+    console.log(data.foundUser);
```
Comment (MarkVed17) [inline src/Services/login-service.js:9]: remove this `console` later.

File: src/Services/login-service.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    return data;
+  } catch (e) {
+    alert('user not found');
+    console.log(e.response.data);
```
Comment (MarkVed17) [inline src/Services/login-service.js]: even this.

File: src/backend/db/users.js [hunk 5]
```diff
@@ -8,11 +8,11 @@ import { formatDate } from "../utils/authUtils";
...
   {
     _id: uuid(),
-    firstName: "Adarsh",
-    lastName: "Balika",
-    username: "adarshbalika",
-    password: "adarshBalika123",
+    firstName: 'Adarsh',
+    lastName: 'Balika',
+    username: 'adarshbalika',
+    password: 'adarshBalika123',
```
Comment (MarkVed17) [inline src/backend/db/users.js:14]: change the dummy username to something else. (if you remember). Better do it in the early stages of building the app itself.

**Review by MarkVed17 — APPROVED** [review]
I like the way you've named the CSS classes. Rest all looks good to proceed.

**Pull request merged**